app/src/css.ts

```diff
@@ -1,5 +1,5 @@
-import fontIndex from './gfonts'
 import { getStyle } from './config'
+import fontIndex from './gfonts'
```

Comment:
Should come before `import { getStyle } …`

Reply:
Fixed